site/app/libraries/database/DatabaseQueries.php

```diff
@@ -9372,6 +9399,44 @@ public function checkNonMappedUsers() {
         return $this->rowsToArray($this->submitty_db->rows());
     }
 
+    /**
+     * Adds a component grader to the current_gradable_grader table
+     *
+     * @param \app\models\gradeable\Component $component
+     * @param bool $isTeam
+     * @param string $grader_id
+     * @param string $graded_id
+     * @return void
+     */
+    public function addComponentGrader($component, $isTeam, $grader_id, $graded_id) {
```

Comment:
It would be better to use native PHP type annotations for these arguments as follows.  PHPdoc annotations are only necessary if you're providing extra information, such as a template type.  Apply throughout.
```suggestion
    public function addComponentGrader(Component $component, bool $isTeam, string $grader_id, string $graded_id): void {
```